app/controllers/users_controller.rb

```diff
@@ -52,6 +52,21 @@ def show
     end
   end
 
+  def edit_password
+    @user = current_user
+  end
+
+  def update_password
+    @user = User.find(current_user.id)
```

Comment:
I believe I was over thinking this when I was trying to guard against a user trying to post request to another user id's password. You're totally correct, this should just be current_user.